src/app/account/account-list/account-list.component.ts

```diff
@@ -29,6 +35,14 @@ export class AccountListComponent {
     };
   }
 
+  public get itemComponent() {
+    if (this.mode === ViewMode.BOX) {
```

Comment:
return this.mode === ViewMode.BOX?AccountCardItemComponent:AccountRowItemComponent;

Reply:
Please do the same changes to the rest of the code